metadata-models/src/main/pegasus/com/linkedin/common/ClientId.pdl

```diff
@@ -0,0 +1,14 @@
+namespace com.linkedin.common
+
+/**
+* A simple wrapper around a String to persist the client ID for telemetry in DataHub's backend DB
+*/
+@Aspect = {
+    "name": "clientId"
+}
+record ClientId {
```

Comment:
Sorry about the renaming comment, but if we are going to use this specific for Telemetry, then I think we should name this TelemetryClientId and create a telemetry package and put it there. We been isolating aspects that bound to specific entities in it's own package (like policy / assertions / etc. )

Reply:
No worries, can do 👍